README.md

```diff
@@ -12,28 +12,22 @@ A framework for creating flexible, powerful admin dashboards in Rails.
 
 ![administrate](https://user-images.githubusercontent.com/11917/72203824-ec10f980-3468-11ea-9ac1-51cd28ff88b7.png)
 
-## What Is Administrate?
-
-Administrate is a library for Rails apps
-that automatically generates admin dashboards.
-Administrate's admin dashboards give non-technical users clean interfaces
-that allow them to create, edit, search, and delete records
-for any model in the application.
-
-Administrate solves the same problem as [Rails Admin] and [ActiveAdmin],
-but aims to provide a better user experience for site admins,
-and to be easier for developers to customize.
-
-To accomplish these goals, Administrate follows a few guiding principles:
-
-- No DSLs (domain-specific languages)
-- Support the simplest use cases, and let the user override defaults with
-  standard tools such as plain Rails controllers and views.
-- Break up the library into core components and plugins,
-  so each component stays small and easy to maintain.
-
-[Rails Admin]: https://github.com/sferik/rails_admin
-[ActiveAdmin]: http://activeadmin.info/
+## What's Administrate?
+
+Administrate is a library for Rails that generates admin dashboards.
+Administrate's admin dashboards give users clean interfaces that allow them to
+create, edit, search, and delete records for any model in the application. It
+aims to provide the best user experience, and doing as much work as possible
+for you, whilst also being easy to customise.
```

Comment:
Instead of easy, you could maybe use "flexible" or just "customisable"?

Reply:
Ooh yeah, "flexible" does actually fit quite well.